sheriff_test.go

```diff
@@ -464,3 +464,32 @@ func TestMarshal_EmbeddedFieldEmpty(t *testing.T) {
 
 	assert.Equal(t, string(expected), string(actual))
 }
+
+type TestInlineStruct struct {
+	tableName struct{} `json:"-"`
```

Comment:
`go vet` says this is an error because json tag in unexported field name. The test must actually test this specific case, right? 

> ./sheriff_test.go:469: struct field tableName has json tag but is not exported
(https://travis-ci.org/liip/sheriff/jobs/297684025)

Seems like we need to switch to specific flags only.. I can handle this later on.